src/databricks/labs/ucx/workspace_access/groups.py

```diff
@@ -975,6 +910,81 @@ def _get_strategy(
+        # The owner group will be assigned by default to all migrated tables/schemas
+        user_id = self._ws.current_user.me().id
+        if not user_id:
+            logger.error("Couldn't find the user id of the current user.")
+            return None
+        groups = self._user_account_groups(user_id)
+        if not groups:
+            logger.warning("No account groups found for the current user.")
+            return None
+        if len(groups) == 1:
+            return groups[0].display_name
+        group_names = [group.display_name for group in groups]
+        return prompt.choice("Select the group to be used as the owner group", group_names, max_attempts=3)
+
+    def validate_owner_group(self, group_name: str) -> bool:
+        # This method is used to validate that the current owner is a member of the group
+        user_id = self._ws.current_user.me().id
+        if not user_id:
+            logger.warning("No user found for the current session.")
+            return False
+        groups = self._user_account_groups(user_id)
+        if not groups:
+            logger.warning("No account groups found for the current user.")
+            return False
+        group_names = [group.display_name for group in groups]
+        return group_name in group_names
+
+    def _user_account_groups(self, user_id: str) -> list[Group]:
+        # This method is used to find all the account groups that a user is a member of.
+        groups: list[Group] = []
+        account_groups = self._list_account_groups("id,displayName,externalId,members")
+        if not account_groups:
+            return groups
+        for group in account_groups:
+            if not group.members:
+                continue
+            for member in group.members:
+                if member.value == user_id:
+                    groups.append(group)
+                    break
+        return groups
+
+    def _list_account_groups(self, scim_attributes: str) -> list[iam.Group]:
+        # TODO: we should avoid using this method, as it's not documented
+        # get account-level groups even if they're not (yet) assigned to a workspace
+        logger.info(f"Listing account groups with {scim_attributes}...")
+        account_groups = []
+        raw = self._ws.api_client.do("GET", "/api/2.0/account/scim/v2/Groups", query={"attributes": scim_attributes})
+        for resource in raw.get("Resources", []):  # type: ignore[union-attr]
+            group = iam.Group.from_dict(resource)
+            if group.display_name in SYSTEM_GROUPS:
+                continue
+            account_groups.append(group)
+        logger.info(f"Found {len(account_groups)} account groups")
+        sorted_groups: list[iam.Group] = sorted(
+            account_groups, key=lambda _: _.display_name if _.display_name else ""
+        )  # type: ignore[arg-type,return-value]
+        return sorted_groups
+
+    def account_groups_in_account(self) -> dict[str, Group]:
```

Comment:
Given the name of the class, the following should be sufficient, can be merged with the optional user referenced in other comment

```suggestion
    def list(self) -> list[Group]
```

Reply:
Changed the name. Too much downstream dependency to change the signature